src/context/GlobalContext.js

```diff
@@ -1,4 +1,4 @@
-import { createContext, useEffect, useReducer } from "react";
+import React, { createContext, useEffect, useReducer } from "react";
```

Comment:
idem aos comentários acima

Reply:
Solução para o erro `'React' must be in scope when using JSX  react/react-in-jsx-scope`.
Vou verificar atualizações para ver se a exigência persiste.